specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/hyperdrive.json

```diff
@@ -0,0 +1,714 @@
+      "x-ms-discriminator-value": "Bandit"
+    },
+    "MedianStoppingPolicy": {
+      "type": "object",
+      "description": "Median stopping policy configuration. Please refer https://docs.microsoft.com/en-us/python/api/azureml-train-core/azureml.train.hyperdrive.medianstoppingpolicy?view=azure-ml-py for more information.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/HyperDrivePolicyConfigBase"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "type": "object",
+              "description": "Policy configuration properties.",
+              "properties": {
+                "evaluation_interval": {
+                  "format": "int32",
+                  "type": "integer",
+                  "description": "Evaluation interval of the policy."
+                },
+                "delay_evaluation": {
+                  "format": "int32",
+                  "type": "integer",
+                  "description": "Value indicating the number of sequences for which the first evaluation is delayed."
+                }
+              }
+            }
+          }
+        }
+      ],
+      "properties": {},
+      "x-ms-discriminator-value": "MedianStopping"
+    },
+    "TruncationSelectionPolicy": {
+      "type": "object",
+      "description": "Truncation selection policy configuration. Please refer https://docs.microsoft.com/en-us/python/api/azureml-train-core/azureml.train.hyperdrive.truncationselectionpolicy?view=azure-ml-py for more information.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/HyperDrivePolicyConfigBase"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "type": "object",
+              "description": "Policy configuration properties.",
+              "properties": {
+                "evaluation_interval": {
+                  "format": "int32",
+                  "type": "integer",
+                  "description": "Evaluation interval of the policy."
+                },
+                "delay_evaluation": {
+                  "format": "int32",
+                  "type": "integer",
+                  "description": "Value indicating the number of sequences for which the first evaluation is delayed."
+                },
+                "truncation_percentage": {
+                  "format": "int32",
+                  "type": "integer",
+                  "description": "Truncation percentage value."
+                },
+                "exclude_finished_jobs": {
+                  "type": "boolean",
+                  "description": "Boolean indicating if metrics from finished jobs should be excluded in the policy decision process."
+                }
+              }
+            }
+          }
+        }
+      ],
+      "properties": {},
+      "x-ms-discriminator-value": "TruncationSelection"
+    },
+    "ChoiceParameterExpression": {
```

Comment:
Is this model being used anywhere?

Reply:
This is provided for format guidelines for parameter_space. Not intended for use.

Reply:
No further action for now.